client/blocks/upload-image/docs/example.jsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { partial } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import UploadImage from '../';
+import { AspectRatios } from 'state/ui/editor/image-editor/constants';
+
+export default class UploadImageExample extends Component {
+	state = {
+		isUploading: false,
+		uploadedImageDataUrl: null
+	};
+
+	render() {
+		const { isUploading, uploadedImageDataUrl } = this.state;
+
+		return (
+			<div className="docs__design-assets-group">
+				<h3>Default Upload Image</h3>
+				<UploadImage
+					isUploading={ isUploading }
+					onImageEditorDone={
+						( imageBlob ) => {
+							this.setState( {
+								uploadedImageDataUrl: URL.createObjectURL( imageBlob ),
+								isUploading: true,
+							} );
+						}
+					}
+					imageEditorProps={ {
+						defaultAspectRatio: AspectRatios.FREE,
+					} }
+				/>
+
+				<h3>Image is uploaded</h3>
+				<UploadImage
+					placeholderContent={ null }
+					uploadingContent={ null }
+				>
+					<img src="https://cldup.com/mA_hqNVj0w.jpg"/>
```

Comment:
Yeah base64 will explode it by quite a bit, but if you find a small test image, it might not matter that much. Another option is that we can maybe point this at a media asset that we serve from our CDN.